cmd/docker/main.go

```diff
@@ -106,6 +106,8 @@ func handleFlags() (*CliOptions, bool) {
 	portLayerAddr := flag.String("port-layer-addr", "127.0.0.1", "Port layer server address")
 	portLayerPort := flag.Uint("port-layer-port", 9001, "Port Layer server port")
 
+	debug := flag.Bool("debug", false, "Enable debuglevel logging")
```

Comment:
hmm how about keeping this and checking whether the debug flag or vchConfig is set? Otherwise we need guestinfo values all the time